.for_maintainers/release/AMI_RELEASE.md

```diff
@@ -0,0 +1,101 @@
+# Releasing the publicly-accessible Wallaroo Amazon Machine Images (AMIs)
+
+This document is aimed at members of the Wallaroo team who might be releasing
+AMIs for a release candidate or release branch.
+
+To learn more about our release process, see [RELEASE.md](RELEASE.md).
+
+## Prerequisites for releasing the AMIs
+
+The AMI building and publishing process is conducted inside a Vagrant
+machine. You'll therefore need:
+
+* Vagrant installed
+* A provisioned Wallaroo Release Vagrant box, as described in
+  [PROVISION_VAGRANT.md](PROVISION_VAGRANT.md).
+* Valid AWS credentials in `~/.aws`
```

Comment:
a link to how to set this up would be handy for those who might not have AWS credentials set up